bin-src/lib/getDependentStoryFiles.ts

```diff
@@ -93,17 +93,28 @@ export async function getDependentStoryFiles(
   const namesById: Record<number, string> = {};
   const reasonsById: Record<number, string[]> = {};
   const csfGlobsByName: Record<string, true> = {};
-
+  const CSF_REGEX = /.\/src.*?sync/g;
```

Comment:
maybe `\s+sync` -- I suspect the whitespace in the string is crucial (and not allowed in normal "urls").

The `./src` part will be project specific so I'd leave that off.